Makefile

```diff
@@ -99,6 +99,10 @@ else
 	bash sanity/check_sanity.sh
 endif
 
+crio:
+	bash .ci/install_bats.sh
+	RUNTIME=${RUNTIME} ./integration/cri-o/cri-o.sh
```

Comment:
Which versions of CRI-O we care about testing?
RUNTIME doesn't work with newer versions, those use CONTAINER_DEFAULT_RUNTIME, IIRC.
CONTAINER_DEFAULT_RUNTIME doesn't work with older versions, though.

So, the best way to go for now is exporting both.

Reply:
Uh, missed it!  I moved the env vars directly in the cri-o.sh script, RUNTIME here was ignored, so removing it!
The vars have shimv2 compatible defaults that will be picked up for CRI-O configuration by the CRI-O test script. The user may still export its own values if needed (exporting CONTAINER_DEFAULT_RUNTIME, RUNTIME_ROOT and RUNTIME_TYPE) but I would leave them out of the Makefile... I don't expect a use case where the user needs to rediefine the vars too. Could this option be viable?


> And the static checks are failing due to:
> 
> ```
> ERROR: Commit 4f783e3bed9a5fcd80bdae2da69e55512635a22c: Failed to find subsystem in subject: "Revert \"integration: Remove old CRI-O tests\""
> ERROR: checkcommits failed. See the document below for help on formatting
> commits for the project.
> 
> https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#patch-format
> ```